receiver/prometheusexec/subprocessmanager/config/config.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
```

Comment:
I'd move this into `receiver/prometheusexec/config.go`

By putting it here you're creating another package people would unnecessarily need to import if they want to instantiate your receiver.

Reply:
Will change, thanks!

Reply:
EDIT: I had put this within `subprocessmanager` initially for context, but also since if it's in the parent package `prometheusexec` I get some cycle import for parts of the `Process` struct in `manager.go`!